bedrock/firefox/templates/firefox/firstrun/firstrun_quantum.html

```diff
@@ -0,0 +1,111 @@
+    </svg>
+
+    <svg class="st4 background" xmlns="http://www.w3.org/2000/svg" viewBox="-270 550 2100 290" preserveAspectRatio="none">
+      <path d="M1784.3 638c-234-191.8-398.9 43.1-824.9 197.8C565 979 307.4 476.2 26.2 564.6c-281.2 88.4-314.8 588 182 913.2s1345.9 140.9 1549.6-140C1915 1121 1949 773 1784.3 638z"/>
+    </svg>
+
+    <svg class="st5 background" xmlns="http://www.w3.org/2000/svg" viewBox="-278 -170 1900 750" preserveAspectRatio="none">
+      <path d="M71.8 503.1c1315 237.8 1451-225.9 1443-438S778.8 405 415.4 282.9c-316.6-106.4-497-380-684.8-337s-973.8 319.4 341.2 557.2z"/>      
+    </svg> 
+
+    <svg class="st6 background" xmlns="http://www.w3.org/2000/svg" viewBox="-300 -150 2050 900" preserveAspectRatio="none">
+      <path d="M495.3-365.1C66.6-148.4-687.9 864.2-325.8 729.7s582.1-353.8 847.5-390c453.8-61.9 814 456.7 1344.2 334.6 530.2-122-579.6-1305.2-1370.6-1039.4z"/>
+    </svg>
+
+    <svg class="st7 background" xmlns="http://www.w3.org/2000/svg" viewBox="-230 -200 1380 790" preserveAspectRatio="none">
+      <path d="M-583.2-65.9c56.5 448.6 1076.5 1063.5 1574.5 248S470.2 26.6 3.8-335.4s-643.6-179.1-587 269.5z"/>
+    </svg>
+
+    <svg class="st8 background" xmlns="http://www.w3.org/2000/svg" viewBox="-155 -80 2000 580" preserveAspectRatio="none">
+      <path d="M1327.5-88.5c-1315-237.8-1451 225.9-1443 438S620.5 9.7 983.9 131.8c316.6 106.4 497 380 684.8 337s973.8-319.4-341.2-557.3z"/>
+    </svg>
+
+    <svg class="white-overlay" viewBox="0 0 1440 800" preserveAspectRatio="none">
+      <rect x="0" y="0" width="1440" height="800"/>
+    </svg>
+
+    <svg class="cls-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 800" preserveAspectRatio="none">
+      <defs>
+        <linearGradient id="SVGID_1" gradientUnits="userSpaceOnUse" x1="765.723" y1="594.394" x2="769.406" y2="229.758" gradientTransform="matrix(1 0 0 -1 0 800)">
+          <stop offset="0" stop-color="#002989"/>
+          <stop offset=".50" stop-color="#0042B4"/>
+          <stop offset="1" stop-color="#005FE7"/>
+        </linearGradient>
+        <linearGradient id="SVGID_2" gradientUnits="userSpaceOnUse" x1="746.076" y1="694.442" x2="665.534" y2="-609.57" gradientTransform="matrix(1 0 0 -1 0 800)">
+          <stop offset="0" stop-color="#005FE7"/>
+          <stop offset=".64" stop-color="#002079"/>
+        </linearGradient>
+        <linearGradient id="linear-gradient" x1="653.02" y1="155" x2="950.05" y2="1256.11" gradientUnits="userSpaceOnUse">
+          <stop offset="0" stop-color="#ff272d"/>
+          <stop offset=".5" stop-color="#ff8e00"/>
+          <stop offset="1" stop-color="#ffde00"/>
+        </linearGradient>
+      </defs>
+      <rect x="0" y="0" width="1440" height="800"/>
+    </svg>
+
+    <div class="fxaccounts-container">
+      {% block page_content %}
```

Comment:
I don't think we need this block, at least not at this point. Looks like it was carried over from the previous page that had a few variant templates for an A/B test, but we don't have such a test for this page yet.